src/udev/udev-builtin-blkid.c

```diff
@@ -58,6 +58,10 @@ static void print_property(sd_device *dev, bool test, const char *name, const ch
                 blkid_encode_string(value, s, sizeof(s));
                 udev_builtin_add_property(dev, test, "ID_FS_LABEL_ENC", s);
 
+        } else if (startswith(name, "FS")) {
```

Comment:
that looks like a pretty broad match... i.e. the word "FSALT" would match this, even though this is not really what is meant here...

The other case where we use a startswith() match here checks for a prefix ending in an underscore, which makes clear we check for a whole namespace (as that's how we do namespacing in udev props/env vars: via underscore). But this check here is super duper generic, just two alphabetic chars...

hence, given you know which fields you want to support here, why not list them here, i.e. use `STR_IN_SET(name, "FSSIZE", "FLSLASTBLOCK", "FSNLOCKSIZE")`?

Reply:
Sure. Initially, I decided to check only for `FS` in case any new tags pop-up. But I agree, it's too general. Changed to suggested macro.